contracts_use_cases.html

```diff
@@ -492,6 +492,17 @@ <h2>Use cases &mdash; table</h2>
   <td>Define and check invariants over all entry and exit points from a coroutine (to its awaiter or promise)</td>
   <td></td></tr>
 
+<tr><td></td>
+  <td>C++ Library Developer</td>
+  <td>Use templates</td>
+  <td>Be able to ship header only library</td>
```

Comment:
This one does not seem to be related to contracts. Maybe "use contracts with templates in order to be able to ship header-only library with contracts.

Reply:
I think it should be implied that everything in the document is related to contracts. It would be a lot more verbose if every sentence used the word "contracts" explicitly.

If you prefer I'll accept 
```
  <td>C++ Library Developer</td>
  <td>Use templates with contracts</td>
  <td>Be able to ship header only library</td>
```

But then why wouldn't you write:
```
  | C++ Developer | Support modern features with contracts | Minimize use of macros with contracts |  
  | C++ Developer | Support modern features with contracts | Contracts to be interoperable with modules |  
  | C++ Developer | Support modern features with contracts | Contracts to be interoperable with coroutines |  
  | C++ Developer | Support modern features with contracts | Contracts to be interoperable with concepts

etc...
```

I'd like to avoid unnecessary redundancy. As it stands we're asking busy people to read and think about a lot of text.

Reply:
Your first suggestion addresses my concern.
The reason I pick only on this one is that it describes something at a higher level: delivering libraries.